components/molecules/FeedbackWidget.js

```diff
@@ -275,6 +275,7 @@ export const FeedbackWidget = ({
                   />
                   <ActionButton
                     id="feedback-submit"
+                    ariaLabel="Submit feedback button"
```

Comment:
Should just be `Submit feedback`, otherwise screen readers will announce `Submit feedback button button`